### ISSUE TITLE
Add onInit option to sandboxFactoryFn

### ISSUE THREAD

**Issue by ertrzyiks** [ertrzyiks/hexo-test-utils#12]
The current signature os the `sandboxFactoryFn` is just one String param with a fixture name
```js
const sandbox = createSandbox()
sandbox('fixture_name')
```

support passing options and the new option `onInit`
```js
const sandbox = createSandbox()
sandbox({ 
  fixture_name: 'fixture_name',
  onInit: [load, process]
})
```

It should be an array of function which should be reduced with the context.

**Comment by ertrzyiks** [comment]
This or equivalent

**Comment by ertrzyiks** [comment]
Fixed by #16
